### PR TITLE
Advanced Fatigue - Fix config macro

### PR DESCRIPTION
**Pull request by PabstMirror**
Without arma crashes with
```
ErrorMessage: File z\ace\addons\advanced_fatigue\CfgEden.hpp, line 5: Config: End of line encountered after 
Application terminated intentionally
```

### CODE REVIEW

**Comment by johnb432** [comment]
> Without arma crashes with
> 
> ```
> ErrorMessage: File z\ace\addons\advanced_fatigue\CfgEden.hpp, line 5: Config: End of line encountered after 
> Application terminated intentionally
> ```

What the hell - it's fine for me.

File: addons/advanced_fatigue/CfgEden.hpp [hunk 1]
```diff
@@ -3,7 +3,7 @@ class Cfg3DEN {
         class Slider;
         class GVAR(slider): Slider {
             #pragma hemtt suppress pw3_padded_arg
-            attributeLoad = QUOTE(
+            attributeLoad = QUOTE( \
```
Comment (johnb432) [inline addons/advanced_fatigue/CfgEden.hpp:6]: Everywhere else we have this structure we don't have a space
```suggestion
            attributeLoad = QUOTE(\
```

**Review by johnb432 — APPROVED** [review]
Addressing the comment is optional, LGTM

**Comment by PabstMirror** [comment]
it will work fine with hemtt as it can parse this macro and then output rapified configs
u can test with `hemtt.exe launch --no-rap` to have arma handle the configs natively and get the error

File: addons/advanced_fatigue/CfgEden.hpp [hunk 2]
```diff
@@ -12,7 +12,7 @@ class Cfg3DEN {
             );
             attributeSave = QUOTE(params ['_ctrlGroup']; sliderPosition (_ctrlGroup controlsGroupCtrl 100));
             #pragma hemtt suppress pw3_padded_arg
-            onLoad = QUOTE(
+            onLoad = QUOTE( \
```
Comment (PabstMirror) [inline addons/advanced_fatigue/CfgEden.hpp:15]:
```suggestion
            onLoad = QUOTE(\
```

**Pull request merged**